### DISCUSSION TITLE
There is a problem after open dev tools on kibana and send the below request.

### DISCUSSION THREAD

**Discussion by xianzheTM** [jertel/elastalert2#758] [Q&A] [answered]
I refer to this document and use chart to install.[chart install](https://github.com/jertel/elastalert2/tree/master/chart/elastalert2)
elasticsearch version: 7.17.
And send the below request:
```
PUT /elastalert/_mapping/elastalert?include_type_name=true
{
  "properties": {
      "alert_time": {"type": "date"}
  }
}
```
Compared with the document, I added `include _ type _ name`.
Here is the response.:
```
{
  "error" : {
    "root_cause" : [
      {
        "type" : "illegal_argument_exception",
        "reason" : "Rejecting mapping update to [elastalert] as the final mapping would have more than 1 type: [_doc, elastalert]"
      }
    ],
    "type" : "illegal_argument_exception",
    "reason" : "Rejecting mapping update to [elastalert] as the final mapping would have more than 1 type: [_doc, elastalert]"
  },
  "status" : 400
}
```
Below is the log in Pod, which looks normal.
![image](https://user-images.githubusercontent.com/25103583/157000320-c44c4328-f374-47a4-a458-b05cf82971d5.png)

**Comment by jertel** [comment, marked as answer]
I believe that suggestion is for users upgrading from older ElastAlert versions to the 6.x and higher. If this is a new installation you shouldn't need to execute that. Are you seeing errors with ElastAlert, which caused you to try that command?

**Reply by xianzheTM** [reply]
> 
Everything is fine except this question. I have been able to receive the alarm. I am just worried that this mistake will lead to some problems. Thank you for your answer.
